nservicebus/messaging/message-correlation.md

```diff
@@ -7,14 +7,18 @@ related:
  - nservicebus/messaging/headers
 ---
 
-Message correlation is the act of connecting instances of request messages with instances of response messages. Since all messages have a `Message Id` the `Correlation Id` is just a reference back to the specific message instance that caused this message to be sent back as a reply.
+Message correlation connects instances of request messages with instances of response messages. Each message has a `Message Id` and the `Correlation Id` is a reference back to the specific message instance that caused this message to be sent back as a reply.
 
 This is a pattern from the [Enterprise Integration Patterns book](http://www.enterpriseintegrationpatterns.com/patterns/messaging/CorrelationIdentifier.html).
 
-One example usage is that it allows NServiceBus to find out which callback to invoke should the user have requested a callback when the request message was sent out.
+One example usage within NServiceBus is to find out which callback to invoke should the user have requested a callback when the request message was sent out.
```

Comment:
@dvdstelt When we publish an event from within the handler, the correlation Id is the message Id of the published message, not the correlation Id of the incoming message. Correct?